src/SFA.DAS.Campaign.Web/Controllers/RegisterInterestController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Campaign.Domain.DataCollection;
+using SFA.DAS.Campaign.Models.DataCollection;
+using SFA.DAS.Campaign.Web.Constants;
+using SFA.DAS.Campaign.Web.Models;
+
+namespace SFA.DAS.Campaign.Web.Controllers
+{
+    public class RegisterInterestController : Controller
+    {
+        private readonly IUserDataCollection _userDataCollection;
+
+        public RegisterInterestController(IUserDataCollection userDataCollection)
+        {
+            _userDataCollection = userDataCollection == null ? throw new NullReferenceException() : userDataCollection ;
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> Index(RegisterInterestModel registerInterest)
+        {
+            if (ModelState.IsValid)
+            {
+                var userData = new UserData()
+                {
+                    FirstName = registerInterest.FirstName,
+                    LastName = registerInterest.LastName,
+                    Email = registerInterest.EmailAddress,
+                    CookieId = registerInterest.ReturnUrl,
```

Comment:
Might be worth setting these to empty string until the correct values are populated. The CookieId should be the unique Id from the GA tracking cookie.
Route Id will be a value for citizen or employer

Reply:
i wanted to do this originally, however the validation on the userdata call doesnt allow blanks so had to populate it with something!